ground/src/main/java/com/google/android/ground/ui/startup/StartupViewModel.kt

```diff
@@ -29,7 +29,10 @@ internal constructor(
 
   /** Checks & installs Google Play Services and initializes the login flow. */
   suspend fun initializeLogin() {
-    googleApiManager.installGooglePlayServices()
+    val isGooglePlayServicesAvailable = googleApiManager.installGooglePlayServices()
+    if (!isGooglePlayServicesAvailable) {
```

Comment:
Same comment here. This entire condition goes away if `installGooglePlayServices` throws an exception.